streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -2011,7 +1883,7 @@ public void shouldReturnAllActiveTasksToPreviousOwnerRegardlessOfBalanceIfEndOff
         createMockTaskManager(allTasks, emptyTasks);
         adminClient = EasyMock.createMock(AdminClient.class);
         expect(adminClient.listOffsets(anyObject())).andThrow(new StreamsException("Should be handled"));
-        configureDefaultPartitionAssignor();
```

Comment:
IIUC, it already is, since you can switch assignors via this config, but maybe I don't understand the implications of your suggestion. The intent of my question was: how many of these tests are specific to the sticky assignment strategy (and should be moved to a new test class specific to the sticky assignor), and how many are testing invariants about the `StreamsPartitionAssignor`, which should hold with both assignors.

Reply:
I just looked at these changes before seeing this comment, and reached the same conclusion. Thanks for taking this on!